build.sbt

```diff
@@ -114,7 +113,7 @@ lazy val docs = project
       "white-color"       -> "#FFFFFF"),
     ghpagesNoJekyll := false,
     micrositeAuthor := "Kailuo Wang",
-    tutScalacOptions ~= (_.filterNot(Set("-Ywarn-unused-import", "-Ywarn-dead-code"))),
+    scalacOptions in Tut ~= (_.filterNot(Set("-Ywarn-unused-import", "-Ywarn-dead-code"))),
```

Comment:
to get `Tut` working properly with scalameta, also need to get rid off the `% Provide` in line 145